tests/e2e.test.js

```diff
@@ -426,13 +426,13 @@ describe('Grid item', () => {
     expect(clipboardValue).toMatch(SVG_REGEX);
   });
 
-  it('is possible to download an icon as SVG', async () => {
-    await expect(page).toClick('a[download][href$="svg"]');
-  });
-
-  it('is possible to download an icon as PDF', async () => {
-    await expect(page).toClick('a[download][href$="pdf"]');
-  });
+  it.each(['download-type-svg', 'download-type-pdf'])(
+    'is possible to download an icon "%s"',
+    async (fileType) => {
+      await expect(page).toClick(`button#${fileType}`);
+      await expect(page).toClick(`a[download].download-type-file`);
```

Comment:
Any particular reason you're using backticks here?

```suggestion
      await expect(page).toClick('a[download].download-type-file');
```